functions.js

```diff
@@ -0,0 +1,9 @@
+const strictEquals = (values) => {
+    if ('a', 'a') {
+        return true;
+    } else if (a, 'a') {
+        return false;
+    }
+}
```

Comment:
The point of a function is to work on data that is passed into it. For this function to be useful you'd expect the caller to pass in the data that you would check for equality. This function wouldn't let me pass in my own values and get an answer. If I wanted to check `strictEquals(5, 5)` or `strictEquals('this', 'that')` I'd want this function to send me the answer for that scenario.

```JavaScript
const strictEqual = (valOne, valTwo) => {
    if(valOne == val2 && typeof valOne == typeof valTwo) {
        return true
    } else {
        return false
    }
}
```